base/linalg/cholesky.jl

```diff
@@ -203,6 +229,19 @@ The same as `cholfact`, but saves space by overwriting the input `A`, instead
 of creating a copy. An `InexactError` exception is thrown if the factorisation
 produces a number not representable by the element type of `A`, e.g. for
 integer types.
+
+**Example**
+
+```jldoctest
+julia> A = [1 2; 2 50]
+2×2 Array{Int64,2}:
+ 1   2
+ 2  50
+
+ julia> cholfact!(A)
+ ERROR: InexactError()
```

Comment:
`cholfact()` has an example with proper usage. the description of the `!`-version just refers to `cholfact()` as well instead of repeting itself, so imo it makes sense to not create duplicate examples, too. The only addition to the description is the mention of the error, which is what this demonstrates.